docs/conf.py

```diff
@@ -32,11 +32,16 @@
 # Add any Sphinx extension module names here, as strings. They can be
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
-# Warning: I installed numpydoc on my local machine to get this to work
-#          but I am not sure if there is supposed to be a requirements
-#          page for the docs section. So you might need to install this
-#          to use sphinx here. ~Colby
-extensions = ['sphinx.ext.autodoc', 'sphinx.ext.mathjax', 'numpydoc']
+extensions = ['sphinx.ext.autodoc', 'sphinx.ext.mathjax',
+              'sphinx.ext.napoleon', 'sphinx.ext.intersphinx']
```

Comment:
The mapping didn't place nicely with `numpydoc`, so I changed it to `sphinx.ext.napoleon` which also does `numpydoc` docstrings, and is included with `sphinx` anyway.

Reply:
That sounds reasonable. I haven't heard of Napoleon before (thanks!), but after a quick search it seems like a solid option.